app/models/broadcast_event/library_complete.rb

```diff
@@ -1,23 +1,27 @@
 # frozen_string_literal: true
-class BroadcastEvent::LibraryComplete < BroadcastEvent # rubocop:todo Style/Documentation
+
+# Event that indicates library prep is complete.
+# It used to be fired (with slightly different data attached) when the multiplexed library tube was passed.
```

Comment:
Because there could be more changes in how it works in future, maybe we could only describe how it currently works? Probably how it worked before is not going to be needed

Reply:
The data up will reflect the old version of the code, up until the time when this goes live - i.e. there will be library_complete events that were fired when the multiplexed library tube was passed. I'm not planning to do any kind of data migration to change these. So I thought it was worth recording somewhere in case it helped with confusion over reports etc?